### ISSUE TITLE
Alpine 3.16 build

### ISSUE THREAD

**Issue by creationix** [DataDog/ddprof#88]
I'm trying to build from source so I can use this in alpine images, but it's failing on an [import](https://github.com/DataDog/ddprof/blob/main/src/pprof/ddprof_pprof.cc#L9):

```c
#include <ddprof/ffi.h>
```

I can't even find a single reference on google for `ddprof/ffi.h` nor can I find it in the git history here.

Here is my Dockerfile for reference:
```dockerfile
FROM --platform=linux/amd64 alpine:3.16 as build-ddprof
RUN apk add curl cmake build-base xz-dev gtest-dev libcap-dev libexecinfo-dev elfutils-dev
RUN curl -L https://github.com/DataDog/ddprof/archive/refs/tags/v0.8.1.tar.gz | tar -xzv
RUN cd ddprof-0.8.1 && mkdir build && cd build && cmake ..
RUN make -C ddprof-0.8.1/build -j $(nprocs)
```

The full error is:

```sh
/ddprof-0.8.1/src/pprof/ddprof_pprof.cc:11:10: fatal error: ddprof/ffi.h: No such file or directory
   11 | #include <ddprof/ffi.h>
      |          ^~~~~~~~~~~~~~
compilation terminated.
```

**Comment by creationix** [comment]
I think I found it.  It appears to be part of https://github.com/datadog/libdatadog

**Comment by r1viollet** [comment]
Yes, would it help if we provided an alpine build ?
The CMake step is meant to pull down all of the dependencies. I need to update the fetch to retrieve the alpine binaries.

**Comment by r1viollet** [comment]
We now have a branch that builds alpine. We will deliver an alpine binary. Thanks for raising this issue.

**Comment by r1viollet** [comment]
This will be included in the upcoming release, if you want an early custom build or want to discuss other requirements, feel free to reach out.

**Comment by r1viollet** [comment]
This is now released and available in v0.9.3.
